Documentation/cli-installation-scenarios.md

```diff
@@ -0,0 +1,237 @@
+- Only HTTPS links are allowed in any online property 
+
+# Channels
+Channels represent a way for users who are getting the CLI to reason about the stability and quality of the bits they are getting. This is one more way for the user to be fully aware of the state the bits that are being installed are in and to set proper expectations on first use. 
+
+The table below outlines the channels:
+
+| Property         	| Description                                                                                                                                                                                                                                                       	|
+|------------------	|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|
+| Dev              	| Unstable bits that are "bleeding edge". Users are not expected to use this channel often, however it is there for those situations when someone needs/wants a feature that hasn't been stabilizied yet. Also, used for internal testing. 	|
+| Beta/Pre-release 	| Pre-release stable builds with known issues and/or known feature gaps. We are OK with users installing these for preview purposes.                                                                                                                                	|
+| Release          	| Actual releases. Most users are encouraged to install these.                                                                                                                                                                                                      	|
+Channels also impact the NuGet packages. Based on the version of the package specified, the user may end up getting different channels. This is described in more details in [Nuget Packages section](#nuget-packages).  
+
+# Dependencies
+.NET Core CLI is built on top of CoreFX and CoreCLR and as such its' dependencies set is defined by the platform that those two combine. Whether or not those dependencies will be installed depends on the installer being used. On Debian, for instance, using `apt-get` will mean that the appropriate dependencies are installed. For OS X using the PKG (installer) dependencies that are not part of OS X will not be installed. So, to summarize: the CLI bundle will not carry native dependencies of CoreFX and CoreCLR with it. 
+
+A list of dependencies can be found on [dependency list](TBD). 
+
+# Upgrades and updates
+**TODO**
+
+# Layout on disk
+**TODO**
+
+# Acquisition vectors
+By "vectors" we are refering here to the way any given user can obtain the installers. In the CLI, there are two main planned vectors:
+
+1. [Getting Started Page](https://aka.ms/dotnetcoregs)
+2. [Repo landing page](https://github.com/dotnet/cli/blob/rel/1.0.0/README.md)
```

Comment:
There are gaps here:
3. Project Build Scripts [I expect many people won't explicitly get dotnet. It will just come down as part of a clone/build workflow]
4. App Store/Galleries [Listing apt-get, homebrew, etc. packages, whether through cmd line or UI]